src/VirtoCommerce.NotificationsModule.Data/Model/NotificationEntity.cs

```diff
@@ -112,5 +112,12 @@ public virtual void Patch(NotificationEntity notification)
                 Templates.Patch(notification.Templates, (sourceTemplate, templateEntity) => sourceTemplate.Patch(templateEntity));
             }
         }
+
```

Comment:
Better replace this to something like that in-place
https://github.com/VirtoCommerce/vc-platform/blob/release/3.0.0/src/VirtoCommerce.Platform.Data/Model/SettingEntity.cs#L82